src/rgw/rgw_acl.h

```diff
@@ -30,6 +30,8 @@
 
 static constexpr char RGW_REFERER_WILDCARD[] = "*";
 
+struct req_state;
```

Comment:
is this obsolete, since you mentioned https://github.com/ceph/ceph/pull/35860/files#diff-2dc651bac332426c2d8a5bd1478d184eR12

Reply:
these will be removed when I completely change my code to support optional_span paramter and every other req_state from class which I added